apps/vite/src/components/calendar/CalendarHeader.tsx

```
@@ -13,73 +22,43 @@ const CalendarHeader: React.FC<{ month: string; year: string }> = ({ month, year
                   id="list"
-                  onClick={() => setView('event')}
+                  onClick={() => setView("event")}
                 >
                   <ListIcon />
                 </button>
               </div>
-              <button className="hidden md:flex py-2 pl-1.5 pr-3 rounded-md bg-gray-50 border border-gray-300 items-center gap-1.5 text-xs font-medium text-gray-900 transition-all duration-500 hover:">
+              <button
+                className="hidden md:flex py-2 pl-1.5 pr-3 rounded-md bg-gray-50 border border-gray-300 items-center gap-1.5 text-xs font-medium text-gray-900 transition-all duration-500 hover:bg-gray-200"
+                onClick={() => setCurrentNepaliDate(new NepaliDate())}
+              >
```

Biome

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)